src/ui/components/Difference/index.jsx

```diff
@@ -2,40 +2,130 @@ import React, { useState, useEffect } from "react";
 import styled from "styled-components";
 import { nanoid } from "nanoid";
 
+import { useNavigate } from "react-router-dom";
```

Comment:
- `P1`
```suggestion
import styled from "styled-components";
import { nanoid } from "nanoid";
import { useNavigate } from "react-router-dom";

```

`useNavigate`는 외부 라이브러리 이므로 위에 `import`문이랑 붙혀주세요!